database/dbconfig_flags.go

```diff
@@ -0,0 +1,75 @@
+// Code generated by go generate; DO NOT EDIT.
+// This file was generated by robots.
+
+package database
+
+import (
+	"encoding/json"
+	"reflect"
+
+	"fmt"
+
+	"github.com/spf13/pflag"
+)
+
+// If v is a pointer, it will get its element value or the zero value of the element type.
+// If v is not a pointer, it will return it as is.
+func (DbConfig) elemValueOrNil(v interface{}) interface{} {
+	if t := reflect.TypeOf(v); t.Kind() == reflect.Ptr {
+		if reflect.ValueOf(v).IsNil() {
+			return reflect.Zero(t.Elem()).Interface()
+		} else {
+			return reflect.ValueOf(v).Interface()
+		}
+	} else if v == nil {
+		return reflect.Zero(t).Interface()
+	}
+
+	return v
+}
+
+func (DbConfig) mustJsonMarshal(v interface{}) string {
+	raw, err := json.Marshal(v)
+	if err != nil {
+		panic(err)
+	}
+
+	return string(raw)
+}
+
+func (DbConfig) mustMarshalJSON(v json.Marshaler) string {
+	raw, err := v.MarshalJSON()
+	if err != nil {
+		panic(err)
+	}
+
+	return string(raw)
+}
+
+// GetPFlagSet will return strongly types pflags for all fields in DbConfig and its nested types. The format of the
+// flags is json-name.json-sub-name... etc.
+func (cfg DbConfig) GetPFlagSet(prefix string) *pflag.FlagSet {
+	cmdFlags := pflag.NewFlagSet("DbConfig", pflag.ExitOnError)
+	cmdFlags.String(fmt.Sprintf("%v%v", prefix, "host"), defaultConfig.DeprecatedHost, "deprecated")
+	cmdFlags.Int(fmt.Sprintf("%v%v", prefix, "port"), defaultConfig.DeprecatedPort, "deprecated")
+	cmdFlags.String(fmt.Sprintf("%v%v", prefix, "dbname"), defaultConfig.DeprecatedDbName, "deprecated")
+	cmdFlags.String(fmt.Sprintf("%v%v", prefix, "username"), defaultConfig.DeprecatedUser, "deprecated")
+	cmdFlags.String(fmt.Sprintf("%v%v", prefix, "password"), defaultConfig.DeprecatedPassword, "deprecated")
+	cmdFlags.String(fmt.Sprintf("%v%v", prefix, "passwordPath"), defaultConfig.DeprecatedPasswordPath, "deprecated")
+	cmdFlags.String(fmt.Sprintf("%v%v", prefix, "options"), defaultConfig.DeprecatedExtraOptions, "deprecated")
+	cmdFlags.Bool(fmt.Sprintf("%v%v", prefix, "debug"), defaultConfig.DeprecatedDebug, "deprecated")
```

Comment:
These are not good... let's add the change above and regenerate to avoid generating these lines